src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5042,16 +5042,30 @@ private void VisitInitializer(IOperation rewrittenTarget, ISymbolInitializerOper
 
         public override IOperation VisitEventAssignment(IEventAssignmentOperation operation, int? captureIdForResult)
         {
-            var instance = operation.EventReference.Event.IsStatic ? null : operation.EventReference.Instance;
-            if (instance != null)
+            IOperation visitedEventReference, visitedHandler;
+            if (operation.EventReference.Kind == OperationKind.EventReference)
```

Comment:
>if (operation.EventReference.Kind == OperationKind.EventReference) [](start = 12, length = 66)

I think parenthesized IEventReferenceOperation should get the same treatment. For simplicity we can simply skip all IParenthesizedOperation at the top of operation.EventReference